lib/lita/handlers/puppet.rb

```diff
@@ -158,8 +158,9 @@ def nodes_with_class(response)
         end
 
         response.reply("#{username}, let me see what I can find in PuppetDB for you.")
-
-        puppet_classes = class_nodes(url, puppet_class)
+        
+        search = class_camel(puppet_class)
+        puppet_classes = class_nodes(url, search)
         if puppet_classes.empty?
           response.reply("There are no nodes with #{puppet_class} class, are you sure its a valid class?")
```

Comment:
Both here and in the success scenario, it might be better behavior to actually show the user what lita searched for, meaning maybe `puppet_class` should be `search`. I'm not opposed to leaving it as-is, but I feel like this would be less ambiguous to a user wonder if they should try the search again with proper capitalization.